sentinelsat/scripts/cli.py

```diff
@@ -137,6 +137,8 @@ def cli(user, password, geometry, start, end, uuid, name, download, sentinel, pr
 
     if geometry is not None:
         search_kwargs['area'] = geojson_to_wkt(read_geojson(geometry))
+        if api.check_query_length(search_kwargs['area']) > 200:
```

Comment:
condition should be `check_query_length() >= 1`

Reply:
I think there may be an error somewhere else then as well. I tested `api.check_query_length()` with the .geojson provided in #289 (which created the original error). And the output was 689.16937531742. Looking in the comments I noticed someone mentioned the number 200 max. I've never worked with .geojson before so would appreciate learning where I went wrong!

Reply:
changed